src/components/organisms/Onboarding/StepActions.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FormEvent, ReactElement } from 'react'
+import Alert from '../../atoms/Alert'
+import Button from '../../atoms/Button'
+import Loader from '../../atoms/Loader'
+
+import styles from './StepActions.module.css'
+
+interface IStepAction {
+  buttonLabel: string
+  buttonAction: (e: FormEvent) => void
+  successMessage?: string
+  loading: boolean
+  completed: boolean
+  loadingMessage?: string
+}
+
+export default function StepActions({
+  buttonLabel,
+  buttonAction,
+  successMessage,
+  loading,
+  completed,
+  loadingMessage
+}: IStepAction): ReactElement {
+  return (
+    <div className={styles.container}>
+      {loading ? (
+        <Loader message={loadingMessage || 'Loading...'} />
+      ) : completed ? (
+        <Alert
+          text={successMessage}
+          state="success"
```

Comment:
nitpick(non-blocking): See the previous comment. Think about using an enum here.